te/te_docker/Dockerfile

```diff
@@ -75,12 +75,14 @@ missingok \n\
 RUN chmod 0644 /etc/logrotate.d/te-logs
 
 #Creating a cron (Running cron job every 2 mins -- logrotate)
+RUN apt-get install -y cron
 RUN echo "*/2 * * * * /usr/sbin/logrotate /etc/logrotate.d/te-logs" > /etc/cron.d/cron_logrotate_te_logs
 RUN chmod 0644 /etc/cron.d/cron_logrotate_te_logs
 RUN crontab /etc/cron.d/cron_logrotate_te_logs
 
 COPY te_docker/requirements.txt /tmp/
-RUN pip3 install --upgrade "pip < 21.0"
+RUN apt install libpq-dev -y
```

Comment:
Why is dev package needed in the image?

Reply:
https://stackoverflow.com/questions/35104097/how-to-install-psycopg2-with-pg-config-error